### PR TITLE
Clarify the imagePullSecret location

### PR DESCRIPTION
**Pull request by clyang82**
Signed-off-by: clyang82 <clyang@cn.ibm.com>

### CODE REVIEW

File: docs/using-hive.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
...
     * Note that there is an additional mode of DNS management where Hive can automatically create delegated zones for approved base domains. i.e. if hive.example.com exists already, you can specify a base domain of cluster1.hive.example.com on your ClusterDeployment, and Hive will create this zone for you, wait for it to resolve, and then proceed with installation. See below for additional info.
   1. Determine what OpenShift release image you wish to install, and possibly what Hive image you want to use to install it.
-  1. Create a Kubernetes secret containing a docker registry pull secret (typically obtained from [try.openshift.com](https://try.openshift.com)).
+  1. Create a Kubernetes secret containing a docker registry pull secret (typically obtained from [Infrastructure Provider](https://cloud.redhat.com/openshift/install/)).
```
Comment (dgoodwin) [inline docs/using-hive.md]: Could you elaborate on the thought process behind this change? I'm not sure why we'd change try.openshift.com to Infrastructure Provider. I'm also not sure if immediately sending people to a login page is preferable to the level above with a little info on OpenShift 4 at try.openshift.com.

Comment (clyang82) [reply docs/using-hive.md]: I cannot find the imagePullSecret from the try.openshift.com. And I think it is not easy for the beginner to find it by try.openshift.com. change it to infrastructure provider, the user should be easy to find the imagepullsecret for different provider. WDYT?

Comment (dgoodwin) [reply docs/using-hive.md]: try.openshift.com links to cloud.redhat.com/openshift/install at the top, but it contains a little more info about what's going on vs just dropping you at a RH login screen, which might be a little confusing to some. 

For infrastructure providers, the tokens from cloud.redhat.com are valid regardless of the underlying infrastructure. Now in theory, other uses of Hive might implement their own pull secret mechanisms, but this link in particular is meant to imply that typically you'd be obtaining this from Red Hat.

I think I'm ok with the link change, but how about we change to:

[Red Hat](https://cloud.redhat.com/openshift/install/)) - or -
[cloud.redhat.com](https://cloud.redhat.com/openshift/install/))

Comment (staebler) [reply docs/using-hive.md]: I disagree. The URL that we want to be providing to all users of OpenShift is https://try.openshift.com/. We should not provide a URIL that bypasses that initial landing page.

Comment (dgoodwin) [reply docs/using-hive.md]: Honestly this is probably correct, apologies @clyang82 but the current form is probably best, try.openshift.com is the landing page that we should be promoting and the link to cloud.redhat.com is quite prominent at the top of the page.

Comment (clyang82) [reply docs/using-hive.md]: That's OK. Thanks @staebler @dgoodwin for your effort. I am OK with this purpose to expose the landing page for all users.

**Pull request closed without merging**